client/my-sites/marketplace/components/reviews-summary/index.tsx

```diff
@@ -18,28 +18,23 @@ export const ReviewsSummary = ( { slug, productName, productType }: Props ) => {
 	const translate = useTranslate();
 	const [ isVisible, setIsVisible ] = useState( false );
 
-	const { data: marketplaceReviews } = useMarketplaceReviewsQuery( {
+	const { data: marketplaceReviewsStats } = useMarketplaceReviewsStatsQuery( {
 		productType,
 		slug,
 	} );
-	// TODO: The averageRating will come from the server. Calculating it here temporarily.
-	let averageRating = null;
+
 	let numberOfReviews = null;
+	let averageRating = null;
+
 	if (
 		isEnabled( 'marketplace-reviews-show' ) &&
-		marketplaceReviews &&
-		! ( 'message' in marketplaceReviews ) &&
-		marketplaceReviews.length > 0
+		marketplaceReviewsStats?.ratings_count !== undefined &&
```

Comment:
Non blocking comment: The type checker is not happy with these lines as it mentions that `marketplaceReviewsStats` could be of type `ErrorResponse` so you can check that the `ratings_count` field exists in `marketplaceReviewsStats`, which also requires to check that `marketplaceReviewsStats` is not undefined. 

![CleanShot 2023-12-14 at 12 46 51@2x](https://github.com/Automattic/wp-calypso/assets/3519124/c0c04322-012d-4699-9c30-f10870c804c8)

Reply:
This was resolved in https://github.com/Automattic/wp-calypso/pull/85361

Reply:
Yes, indeed, no more type errors 🎉 
![CleanShot 2023-12-18 at 13 12 35@2x](https://github.com/Automattic/wp-calypso/assets/3519124/05af85b6-05e1-44e8-9782-870ff3e5247c)